README.md

```diff
@@ -73,7 +75,7 @@ Snapshots of the development version are available in [Sonatype's snapshots repo
 
 The configuration for an Android project is very similar to the Java case, with one key difference: The `com.google.code.findbugs:jsr305:3.0.2` dependency can be removed; you can use the `android.support.annotation.Nullable` annotation from the Android Support library.
 
-```gradle
+```
```

Comment:
why this change?

Reply:
you know, I don't actually know how this happened, I'm questioning whether githubs web editor removed it, as an unrecognized extension, really it should probably be `groovy` or `kts`